ext/openssl/ossl_x509revoked.c

```diff
@@ -151,6 +159,10 @@ ossl_x509revoked_set_serial(VALUE self, VALUE num)
     return num;
 }
 
+/*
+ * call-seq:
+ *    revoked.time => time
```

Comment:
This method returns `OpenSSL::ASN1::GeneralizedTime` or `OpenSSL::ASN1::UTCTime` depending on the date.